guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/GuvnorM2RepositoryTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2015 JBoss Inc
```

Comment:
Won't. Not sure how much they matter anyway when moving files or cherry-picking.